daphne/src/roles_test.rs

```diff
@@ -318,3 +366,70 @@ async fn http_post_collect_unauthorized_request() {
         Err(DapAbort::UnauthorizedRequest)
     );
 }
+
+#[test]
+fn hpke_decrypter() {
```

Comment:
In tests that use a mock object, usually their methods are not invoked directly, but by other piece of code. Otherwise, this only tests that the mock was well-implemented, but not that is does it's job when used by other party.

Reply:
The test code I wrote there is based on the other tests in the same file.
MockAggregator is not used by other code yet, so I would need to add some code that uses MockAggregator.